src/app/pages/reports-dashboard/components/report/report.component.html

```diff
@@ -114,9 +111,9 @@ <h4 *ngIf="currentEndDate" class="legend-timestamp">
         <section #legendElement class="legend" fxFlex="420px" fxLayout="column" fxLayoutAlign="start">
           <div *ngIf="report && data?.legend" class="legend-timestamp" fxFlex="32px" fxLayoutAlign="start center">
             <h4 *ngIf="data && legendData?.xHTML && shouldShowLegendValue" fxLayoutGap="8px" fxLayoutAlign="start center">
-              <strong>{{ 'Time' | translate }}:</strong>
+              <strong>{{ 'Time22' | translate }}:</strong>
```

Comment:
👆🏼 
(And you need to pull translations from master to recover translations for "Time" that got deleted)